src/content/en/2022/pwa.md

```diff
@@ -1,22 +1,517 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: PWA
-#TODO - Review and update chapter description
-description: PWA chapter of the 2022 Web Almanac covering service workers (usage and features), Web App Manifests, Lighthouse insights, service worker libraries (including Workbox), Web Push notifications and distribution.
+description: PWA chapter of the 2022 Web Almanac covering service workers (usage and features), Web App Manifests, Lighthouse insights, service worker libraries (including Workbox), and Web Push notifications.
 authors: [diekus, Suzzicks]
 reviewers: [aarongustafson, webmaxru, Schweinepriester, tropicadri, beth-panx, tropicadri]
 analysts: [beth-panx]
 editors: [siwinlo]
```

Comment:
@diekus can you go through these lists of contributors and remove anyone who hasn't contributed and also ensure that they're sorted in rough order of most significant contributors first?

Reply:
done.